indico/modules/events/abstracts/forms.py

```diff
@@ -124,6 +127,43 @@ def data(self):
         return data
 
 
+class AbstractJudgmentForm(IndicoForm):
+    """Form for judging an abstract"""
+
+    judgment = IndicoEnumSelectField(_("Judgment"), [DataRequired()], enum=AbstractAction,
+                                     skip={AbstractAction.change_tracks})
+    accepted_track = QuerySelectField(_("Track"), [HiddenUnless('judgment', AbstractAction.accept)],
+                                      get_label=lambda x: x.title, allow_blank=True,
+                                      description=_("The track this abstract will be accepted in"))
+    accepted_contrib_type = QuerySelectField(_("Contribution type"), [HiddenUnless('judgment', AbstractAction.accept)],
+                                             get_label=lambda x: x.name.title(), allow_blank=True,
+                                             description=_("The type of contribution that will be created"))
+    merged_into = AbstractField(_("Merge into"), [HiddenUnless('judgment', 'merge'), DataRequired()],
+                                description=_("The current abstract will be merged onto the selected one"))
+    merge_persons = BooleanField(_("Merge persons"), [HiddenUnless('judgment', AbstractAction.merge), DataRequired()],
+                                 description=_("Authors and speakers of the current abstract will be added to the "
+                                               "selected one"))
+    duplicate_of = AbstractField(_("Duplicate of"),
+                                 [HiddenUnless('judgment', AbstractAction.mark_as_duplicate), DataRequired()],
+                                 description=_("The current abstract will be marked as duplicate of the selected one"))
+    judgment_comment = TextAreaField(_("Comment"), render_kw={'placeholder': _("Leave a comment for the submitter...")})
+
+    def __init__(self, *args, **kwargs):
+        abstract = kwargs.pop('abstract')
+        self.event = abstract.event_new
+        candidate_tracks = abstract.candidate_tracks
```

Comment:
I would expect the property to return a set since there's no useful order. You'd have to convert to list here though so you can still use `[0]` then...